docs/source/algorithms/LoadISISNexus-v2.rst

```diff
@@ -14,13 +14,118 @@ Loads a Nexus file created from an ISIS instrument.
+| (data)         | ``nchannels``                                   |
++----------------+-------------------------------------------------+
+| (data)         | ``nperiods``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[0]``    | ``dur``                                         |
++----------------+-------------------------------------------------+
+| ``IRPB[1]``    | ``durunits``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[2]``    | ``dur_freq``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[3]``    | ``dmp``                                         |
++----------------+-------------------------------------------------+
+| ``IRPB[4]``    | ``dmp_units``                                   |
++----------------+-------------------------------------------------+
+| ``IRPB[5]``    | ``dmp_freq``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[6]``    | ``freq``                                        |
++----------------+-------------------------------------------------+
+| ``IRPB[7]``    | ``gd_prtn_chrg``                                |
++----------------+-------------------------------------------------+
+| ``RRPB[9]``    | ``goodfrm``                                     |
++----------------+-------------------------------------------------+
+| ``RRPB[10]``   | ``rawfrm``                                      |
++----------------+-------------------------------------------------+
+| ``RRPB[11]``   | ``dur_wanted``                                  |
++----------------+-------------------------------------------------+
+| ``RRPB[12]``   | ``dur_secs``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[13]``   | ``mon_sum1``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[14]``   | ``mon_sum2``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[15]``   | ``mon_sum3``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[21]``   | ``rb_proposal``                                 |
++----------------+-------------------------------------------------+
+
+In the Nexus column, groups in capitals are in ``raw_data_1/isis_vms_compat`` and the other groups are in ``raw_data_1``.
+
+(data) indicates that the number is got from the histogram data in an appropiate manner.
+
+``IRPB`` and ``RRPB`` point to the same data. In ``IRPB``, the data is seen as integer and in RRPB it is seen as real (double precision).
+In all cases, integers are passed. Real numbers are used only to allow the integer to be larger.
```

Comment:
If I understood this correctly, I think that IRPB and RRPB do not always have the same data. See for example what I get with HDFView for MER18972: I also see differences in IMAT and ENGINX run files.

![hdfviewer_mer18972](https://cloud.githubusercontent.com/assets/5257172/13568933/cb758c3c-e45c-11e5-8a60-04351a5f7a39.PNG)